b2sdk/bucket.py

```diff
@@ -339,9 +342,11 @@ def ls(
         :param folder_to_list: the name of the folder to list; must not start with "/".
                                Empty string means top-level folder
         :param latest_only: when ``False`` returns info about all versions of a file,
-                              when ``True``, just returns info about the most recent versions
+                            when ``True``, just returns info about the most recent versions
         :param recursive: if ``True``, list folders recursively
         :param fetch_count: how many entries to return or ``None`` to use the default. Acceptable values: 1 - 10000
+        :param with_wildcard: Accepts "*", "?", "[]" and "[!]" in folder_to_list, similarly to what shell does.
+                              If ``True``, automatically assumes recursive was enabled
```

Comment:
I think it should be "As of version 1.18.0 it can only be enabled when `recursive` is enabled" and then we add an assertion, a test, and we make the caller responsible for calling the function with proper arguments.

not forcing recursive to be enabled might be implemented in the future, but for now we just don't support it

Reply:
`1.18.1`, right? Because `1.18.0` is already released. Other than that: 👍